### ISSUE TITLE
Test Individual List Page Interactive Elements for Accessibility

### ISSUE THREAD

**Issue by bpmcneilly** [internetarchive/openlibrary#4902]
# Page to Test:

Individual List - https://openlibrary.org/people/mekBot/lists/OL103955L/Childhood_Favorites

# Steps

Steps for this task are outlined in #4897
 
# Expectations

Test the specified page according to the steps outlined in the parent issue. Create individual bugs with steps to reproduce for any found violations on this page. Reference those bugs on this tracking issue.

Once the page has been tested & all bugs are found, close this issue.

**Comment by RayBB** [comment]
@bpmcneilly I will take on this page. I am not experienced in accessibility testing and would welcome someone checking my work.

**Comment by RayBB** [comment]
This also was completed, actually by me, quite some time ago.
